merge_rp_launches.py

```diff
@@ -0,0 +1,61 @@
+import requests, os, time
+from datetime import datetime
+URL = "https://test.tools.dhis2.org/reportportal/api/v1/dhis2_auto"
+REPORT_PORTAL_TOKEN = os.getenv('RP_TOKEN')
+CI_BUILD_ID = os.getenv('CI_BUILD_ID')
```

Comment:
These lines would fail, if the env vars are not found before going to the following `if` statement.

Reply:
No, it doesn't fail. getenv defaults to None if env vars are not found, so it will reach the following if statement.